lisp/ess-r-mode.el

```diff
@@ -934,22 +929,22 @@ use \"bin/Rterm.exe\"."
 
 ;;;###autoload
 (defun Rnw-mode ()
-  "Major mode for editing Sweave(R) source.
+"Major mode for editing Sweave(R) source.
 See `ess-noweb-mode' and `R-mode' for more help."
-  (interactive)
-  (require 'ess-noweb);; << probably someplace else
-  (setq ess--make-local-vars-permanent t)
-  (ess-noweb-mode 1); turn it on
-  (ess-noweb-set-doc-mode 'latex-mode)
-  (ess-noweb-set-code-mode 'R-mode)
-  (setq ess--local-handy-commands
-        (append '(("weave"      . ess-swv-weave)
-                  ("tangle"     . ess-swv-tangle))
-                ess-handy-commands)
-        ess-dialect "R"
-        ess-language "S")
-  (put 'ess--local-handy-commands 'permanent-local t)
-  (run-mode-hooks 'Rnw-mode-hook))
+(interactive)
+(require 'ess-noweb);; << probably someplace else
```

Comment:
Looks like something is wrong with indentation here.

Reply:
Indeed. Not sure what happened there but it's fixed now. Thanks.